inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/max_pool.cpp

```diff
@@ -0,0 +1,80 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vector>
+
+#include "single_layer_tests/max_pool.hpp"
+#include "common_test_utils/test_constants.hpp"
+
+using namespace LayerTestsDefinitions;
+
+namespace {
+
+std::vector<MaxPoolSpecificParams> max_pool_only_test_cases = {
+        MaxPoolSpecificParams{ { 1, 3, 32, 32 }, { 1, 1 },
+                               { 0, 0 }, { 0, 0 }, { 2, 2 },
+                               ngraph::op::RoundingType::FLOOR, ngraph::op::PadType::SAME_LOWER},
+        MaxPoolSpecificParams{ { 1, 3, 32, 32 }, { 1, 1 },
+                               { 0, 0 }, { 0, 0 }, { 2, 2 },
+                               ngraph::op::RoundingType::FLOOR, ngraph::op::PadType::SAME_UPPER},
+        MaxPoolSpecificParams{ { 1, 3, 32, 32 }, { 1, 1 },
+                               { 0, 0 }, { 0, 0 }, { 2, 2 },
+                               ngraph::op::RoundingType::CEIL, ngraph::op::PadType::SAME_UPPER},
```

Comment:
Should we add tests for other PadType (EXPLICIT, SAME_LOWER, ...)?

Reply:
You are right. I will add it